version: '3.3'

services:
  server-dev:
    image: postgres:12.2
    container_name: va-prod-postgres
    volumes:
      - ./init-db.sql:/docker-entrypoint-initdb.d/init.sql
      - ./restore-db.sh:/docker-entrypoint-initdb.d/restore-db.sh
      - ./valtionavustukset.backup:/var/tmp/valtionavustukset.backup
    ports:
      - '42042:5432'
    command: ["postgres", "-c", "log_statement=all"]
    healthcheck:
      test: ["CMD-SHELL", "pg_isready -U postgres"]
      interval: 10s
      timeout: 5s
      retries: 5
    environment:
      POSTGRES_USER: va
      POSTGRES_PASSWORD: va
      PGDATA: '/tmp/'
      TZ: "Europe/Helsinki"

